tests/main/searching/task.yaml

```diff
@@ -49,7 +49,19 @@ execute: |
     # TODO: discuss with the store how we can make this test stable, i.e.
     #       that section/snap changes do not break us
     if [ "$(uname -m)" = "x86_64" ]; then
-        snap find --section=photo-and-video vlc | MATCH vlc
+        set +e
+        snap find --section=photo-and-video vlc >vlc.log 2>&1
+        retval=$?
+        set -e
+
+        if [ "$retval" -eq 0 ]; then
+          MATCH vlc < vlc.log
+        else
+          # The first case is when load balancer drops the request
+          # The second case (unverified) is when the store is configured to drop the request
+          MATCH 'error: cannot get snap sections: cannot sections: got unexpected HTTP status code 403 via GET to "https://api.snapcraft.io/api/v1/snaps/sections"' < vlc.log \
+          || MATCH 'Request forbidden by administrative rules' < vlc.log
```

Comment:
This is only visible in journalctl -u snapd fwiw

Reply:
Oh, I see. Hmm

Reply:
I've decided not to do this. This addresses the immediate error we see in the store now. We can iterate later, this should help to tame master.